pages/admin/usersActive/components/AccountProfileDetails.js

```diff
@@ -0,0 +1,167 @@
+import { useState, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  Divider,
+  Grid,
+  TextField,
+} from "@mui/material";
+import { useSnackbar } from "notistack";
+import { useRouter } from "next/router";
+import axiosClient from "../../../../axiosClient";
+const states = [
+  {
+    value: "ACTIVE",
+    label: "ACTIVE",
+  },
+  {
+    value: "PENDING",
+    label: "PENDING",
+  },
+];
```

Comment:
If this is a global use constant (i.e. can be used in other places), you should move this to ~/shared/constants.js